docs/cache.txt

```diff
@@ -0,0 +1,240 @@
+.. _laravel-queues:
+
+===============
+Cache and Locks
+===============
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: php framework, cache, lock, code example
+
+Configuration
+-------------
+
+To use MongoDB as a backend for `Laravel Cache and Locks <https://laravel.com/docs/{+laravel-docs-version+}/cache>`__,
```

Comment:
Does `{+laravel-docs-version+}` vary between branches of our own docs, or is it just defined in one place to always refer to the latest version?

I'm not sure why we'd want to link to old versions, as we generally don't do that with the MongoDB server docs. And I also don't see how sourcing the Laravel docs version from a configuration value helps with checking links (in the event a URL is renamed or removed).